### DISCUSSION TITLE
Allow custom Alt+Letter hotkeys to override menu bar hotkeys

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1512] [Suggestions]
Currently the Menu Bar uses hotkeys "Alt+Letter" by default.
F - Files
M - Mark
C - Command
etc.

Would it be possible to allow these hotkeys to be used for other commands, allowing them to override the Menu Bar dropdowns?
So for example, if Alt + F was created as custom hotkey then it would not open the “Files” menu bar dropdown.
I believe all the other letters NOT associated with the Menu Bar can be used as hotkeys, this could be helpful to allow all of them to be customized, allowing access to the full "Alt+Letter" group of hotkeys.

I do not really use the Menu Bar hotkeys, but if this above suggestion is added, it might be a good idea to add a new DC based cmd to apply focus to the Menu Bar (starting focus at the "Files" category at the beginning of the Menu Bar)  This way if a user uses up all the currenlty assigned hotkey letters they could still have access to the Menu Bar via a keyboard shortcut.

Hopefully others will find this suggestion useful
Thank You to anyone who reads this

**Comment by alexx2000** [comment]
It is impossible under Qt. We can disable menu shortcuts only totally.

> to add a new DC based cmd to apply focus to the Menu Bar

Impossible, Lazarus does not have a such capability.

**Comment by advice2020** [comment]
@alexx2000 
Thank you for your response, I apologize I didn't know this was not possible.
My current solution on my own system was to just turn off the menu bar completely which allows the Alt+Letters to be used without opening the menu bar drop downs, which works.
I don't see disabling the shortcuts totally as being a setting that is going to be added to DC, so I will close this post.
Thank You again for your response.
